packages-web/maps/src/Maps.xml

```diff
@@ -11,198 +11,203 @@
                 </property>
                 <property key="apiKeyExp" type="expression" required="false">
                     <caption>API Key</caption>
-                    <description>Please insert your API Key</description>
+                    <description>API Key for usage of the map through the selected provider.</description>
                     <returnType type="String" />
                 </property>
                 <property key="geodecodeApiKey" type="string" required="false">
                     <caption>Google maps API Key</caption>
-                    <description>Please insert your Google API Key for address translation (Geodecoding)</description>
+                    <description />
                 </property>
                 <property key="geodecodeApiKeyExp" type="expression" required="false">
                     <caption>Google maps API Key</caption>
-                    <description>Please insert your Google API Key for address translation (Geodecoding)</description>
+                    <description />
                     <returnType type="String" />
                 </property>
-                <property key="zoom" type="enumeration" defaultValue="automatic">
-                    <caption>Zoom</caption>
-                    <description>Zoom level for single marker, no marker or when fit to markers is disabled.
-                    </description>
-                    <enumerationValues>
-                        <enumerationValue key="automatic">Automatic</enumerationValue>
-                        <enumerationValue key="world">World</enumerationValue>
-                        <enumerationValue key="continent">Continent</enumerationValue>
-                        <enumerationValue key="city">City</enumerationValue>
-                        <enumerationValue key="street">Street</enumerationValue>
-                        <enumerationValue key="buildings">Buildings</enumerationValue>
-                    </enumerationValues>
-                </property>
                 <property key="showCurrentLocation" type="boolean" defaultValue="false">
-                    <caption>Show current location</caption>
-                    <description>Shows the user current location marker</description>
+                    <caption>Show current location marker</caption>
+                    <description>Shows the user current location marker.</description>
                 </property>
             </propertyGroup>
         </propertyGroup>
         <propertyGroup caption="Controls">
             <propertyGroup caption="General">
                 <property key="optionDrag" type="boolean" defaultValue="true">
                     <caption>Drag</caption>
-                    <description>Enable moving center by dragging</description>
+                    <description>The center will move, when end-users drag the map.</description>
```

Comment:
Nitpicking but why there is a , between move and when :)

Reply:
I would say ask Maria :P